AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.m

```diff
@@ -147,22 +147,24 @@ + (void)saveToken:(MSTokenResult *)tokenResult {
  * Get back the Cached Cosmos DB token
  * If the DB token has expired, that token is deleted from KeyChain
  */
-+ (MSTokenResult *_Nullable)retrieveCachedToken:(NSString *)partitionName {
-  if (partitionName) {
-    NSString *tokenString = [MSKeychainUtil stringForKey:[MSTokenExchange tokenKeyNameForPartition:partitionName]];
++ (MSTokenResult *_Nullable)retrieveCachedToken:(NSString *)partition expiredTokenIncluded:(BOOL)expiredTokenIncluded {
+  if (partition) {
```

Comment:
is empty but not `nil` NSString counted as `NO` here? If not - either this might be omitted, or param can be declared as nullable